### DISCUSSION TITLE
Location for installer?

### DISCUSSION THREAD

**Discussion by eddriesen** [koechlm/Vault-Job-Processor---iLogic-Extension#24] [Ideas]
Hi Markus,

We're in the middle of a transition to Vault PRO and I would like to run some ilogic code that used to be local, on the dedicated job processor. 
The youtube tutorials, and readme however don't state where the extension should be installed. On the client system/ dedicated job processor system / vault server or a combination ? 

Thanks for making the extension available! I'm really looking forward to dive into the matter!

**Comment by koechlm** [comment]
Hello,
thank you for bringing this question to my attention; I will update the readme accordingly.
There are two options: 
1) You as an Administrator create iLogic-Jobs for lifecycle transition submissions only: install the extension on the Job-Processor machine only.
2) You as an Administrator create iLogic-Jobs for user-interactive submission: install the extension on any Vault Client. Non-Configuration admin users will be able to use the workflow select one to many files and submit an iLogic-Job, that you as an Admin provided and registered for user consumption.

I hope this helps.

**Reply by eddriesen** [reply]
This saves me a huge amount of time! Many, many thanks!
